src/function_lua.c

```diff
@@ -117,6 +119,7 @@ static int luaEngineCreate(void *engine_ctx, functionLibInfo *li, sds blob, size
         luaErrorInformationDiscard(&err_info);
         goto done;
     }
+    luaGC(lua, &gc_count);
```

Comment:
We will not run this or error?

Reply:
done with [`5c6760b` (#13407)](https://github.com/redis/redis/pull/13407/commits/5c6760b7cb444e5cfa092673e59407859336d4c4)